### PR TITLE
Add support for accessibility identifiers

### PR DESCRIPTION
**Pull request by algrubb**
🛠 Fixes:
* Inability to add/specify accessibility identifiers.

ℹ️ Changes:
* Add support for specifying accessibility identifiers for sections and rows.

### CODE REVIEW

File: Static/Section.swift [hunk 1]
```diff
@@ -43,6 +43,9 @@ public struct Section: Hashable, Equatable {
 
     // MARK: - Properties
 
+    /// The section's accessibility identifier.
+    public var accessibilityIdentifier: String?
```
Comment (eliperkins) [inline Static/Section.swift]: Where is this variable used? It looks like we assign an accessibility identifier to a each cell, but this one for the whole section looks unused.

File: Static/Row.swift [hunk 2]
```diff
@@ -145,9 +148,10 @@ public struct Row: Hashable, Equatable {
 
     // MARK: - Initializers
 
-    public init(text: String? = nil, detailText: String? = nil, selection: Selection? = nil,
+    public init(accessibilityIdentifier: String? = nil, text: String? = nil, detailText: String? = nil, selection: Selection? = nil,
```
Comment (eliperkins) [inline Static/Row.swift]: What do you think about moving this parameter to the end? 

I think for "future compatibility" sake, adding optional parameters to the end of a series of initializers clarifies usage for those upgrading from one version to the next.

Comment (algrubb) [reply Static/Row.swift]: Great point, thanks!

File: Static/Section.swift (hunk 1 above)
Comment (algrubb) [reply Static/Section.swift]: Yeah good call out - it is unused. I've been trying to figure out how to apply this identifier to the section header/footer but am having a bit of trouble trying to implement it. I think for now I'll leave this off and revisit it later.

**Comment by algrubb** [comment]
Ready for another look at your leisure @eliperkins . Removed accessibilityIdentifier from `Section` and added to tests and the example app.

**Pull request merged**